packages/checkout/components/text-input/validated-text-input.tsx

```diff
@@ -211,6 +216,8 @@ const ValidatedTextInput = ( {
 			} }
 			ariaDescribedBy={ describedBy }
 			value={ value }
+			title=""
```

Comment:
This prevents the same error being shown on hover.

Reply:
Will add that as a local comment.